presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoConnection.java

```diff
@@ -547,6 +549,11 @@ public String getSchema()
         return schema.get();
     }
 
+    public Properties getConnectionProperties()
```

Comment:
I would make a defensive copy here, search for examples on defensive copy for Properties objects.